tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -5524,171 +5524,145 @@ auto get_concat_nodedef = [](DataType dtype, int num_inputs) -> NodeDef {
-      InitTestVector<CType>(6),
-      InitTestVector<CType>(6, /*start_value=*/CType(6))};
-  // TODO(hinsu): Use std::vector instead of an array to avoid use of explicit
-  // size.
-  std::vector<TestParams> ok_params = {
+  const std::vector<std::vector<int>> common_input{InitTestVector<int>(6),
+                                                   InitTestVector<int>(6, 6)};
+
+  std::vector<TestParams> params = {
       {
-          /*input_shapes=*/{{1, 2, 3}, {1, 2, 3}},
+          /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
           /*input_values=*/common_input,
           /*axis=*/1,
-          /*expected_output_dims=*/{2, 2, 3},
-          /*expected_output=*/InitTestVector<CType>(12),
+          /*expected_output_dims=*/{1, 2, 2, 3},
+          /*expected_output=*/InitTestVector<int>(12),
       },
       {
-          /*input_shapes=*/{{1, 2, 3}, {1, 2, 3}},
+          /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
           /*input_values=*/common_input,
           /*axis=*/2,
-          /*expected_output_dims=*/{1, 4, 3},
-          /*expected_output=*/InitTestVector<CType>(12),
+          /*expected_output_dims=*/{1, 1, 4, 3},
+          /*expected_output=*/InitTestVector<int>(12),
       },
       {
-          /*input_shapes=*/{{1, 2, 3}, {1, 2, 3}},
+          /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
           /*input_values=*/common_input,
           /*axis=*/3,
-          /*expected_output_dims=*/{1, 2, 6},
+          /*expected_output_dims=*/{1, 1, 2, 6},
           /*expected_output=*/
-          {CType(0), CType(1), CType(2), CType(6), CType(7), CType(8), CType(3),
-           CType(4), CType(5), CType(9), CType(10), CType(11)},
+          {0, 1, 2, 6, 7, 8, 3, 4, 5, 9, 10, 11},
       },
       {
-          /*input_shapes=*/{{1}, {2}, {3}, {1}, {1}, {2}},
+          /*input_shapes=*/{{1, 1}, {1, 2}, {1, 3}, {1, 1}, {1, 1}, {1, 2}},
           /*input_values=*/
-          {{CType(1)},
-           {CType(2), CType(3)},
-           {CType(4), CType(5), CType(6)},
-           {CType(7)},
-           {CType(8)},
-           {CType(9), CType(10)}},
+          {{1}, {2, 3}, {4, 5, 6}, {7}, {8}, {9, 10}},
           /*axis=*/1,
-          /*expected_output_dims=*/{10},
+          /*expected_output_dims=*/{1, 10},
           /*expected_output=*/
-          InitTestVector<CType>(10, /*start_value=*/CType(1)),
+          InitTestVector<int>(10, /*start_value=*/1),
       },
-  };
+      {// An input is a weight
+       /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
+       /*input_values=*/common_input,
+       /*axis=*/1,
+       /*expected_output_dims=*/{1, 2, 2, 3},
+       /*expected_output=*/InitTestVector<int>(12),
+       errors::Unimplemented("The input \"values_1\" for ConcatV2 "
+                             "must be a tensor, at my_concat"),
+       Status::OK(),
+       /*input_as_weight*/ true},
+      {// Axis is batch dimension, should fail in implicit batch mode.
```

Comment:
Similar here, let's move the comment to a new line.

Reply:
Fixed.